bundlesize.config.json

```diff
@@ -10,7 +10,7 @@
     },
     {
       "path": "./packages/instantsearch.js/dist/instantsearch.production.min.js",
-      "maxSize": "74.50 kB"
+      "maxSize": "75 kB"
```

Comment:
is it this change causing the increase or something from the base branch? if so the commit probably could be moved (no big deal if it's squashed later of course)

Reply:
Yeah it's caused by the base branch, I'll put it there once we merge this then